pkg/frontend/transport/handler.go

```diff
@@ -222,6 +226,17 @@ func (f *Handler) reportQueryStats(r *http.Request, queryString url.Values, quer
 		"split_queries", stats.LoadSplitQueries(),
 	}, formatQueryString(queryString)...)
 
+	if queryErr != nil {
+		logMessage = append(logMessage, []interface{}{
+			"status", "failed",
+			"err", queryErr,
+		})
```

Comment:
we want to append these values as individual values, not within a slice, right? (same for `else` condition)

```suggestion
		logMessage = append(logMessage,
			"status", "failed",
			"err", queryErr,
		)
```

Adding assert `assert.Contains(t, strings.TrimSpace(logs.String()), "status=failed")` to the test shows that this is currently broken.

Reply:
Good catch, initially intended to append a slice to slice but forgot the `...` i.e
```
		logMessage = append(logMessage, []interface{}{
			"status", "failed",
			"err", queryErr,
		}...)
```
Will update to individual values like you suggested as it looks a bit cleaner